### ISSUE TITLE
Weird URL escaping behaviour, messed up query args with channels enabled.

### ISSUE THREAD

**Issue by darklow** [andrewgodwin/channels#135]
When I am using pure Django, when I submit a search form with two words and escape it URL looks following:
`/search/one%20two`
`/search/voil%C3%A0` (with unicode string)

In Django I have following `urls.py` and `SearchView` and template:
```py
url(r'^search/(?P<q>(.*))$', search_view)

def search_view(request, q):
        return { 'q': q }
```
Jinja2 template:
```html
Search results for "{{ q|e }}"
```

When using just Django output with spaces or with unicode chars are:
```
Search results for "one two"
Search results for "voilà"
```

But as soon as I add `channels` to `INSTALLED_APPS`
Same output now is:
```
Search results for "one%20two"
Search results for  "voil%C3%A0"
```

And I can't find a way to fix this.
It also affects my search because search string received from arguments is now different, that what user entered.
Could you have any ideas how channels make this different and how to solve problem?
Thank you.

**Comment by andrewgodwin** [comment]
Please tell me the versions you are using of the following packages:

* channels
* django
* daphne
* asgi-redis
* asgiref

That'll help me work out what's going on.

**Comment by darklow** [comment]
Versions:
```
channels==0.11.0
Django==1.9.5
daphne==0.10.3
asgi-redis==0.10.0
asgiref==0.11.2
```

**Comment by andrewgodwin** [comment]
Just released the fix in `channels` `0.12.0`. You'll need to upgrade `daphne` to the most recent version (`0.11.1`) too, though this is specified as a dependency of the new `channels` release.

**Comment by darklow** [comment]
Now it is works as expected.
Thank you for fast response and fix!
